src/components/ForecastDetails.js

```diff
@@ -0,0 +1,26 @@
+import "../styles/ForecastDetails.css";
+
+function ForecastDetails({ forecast }) {
+  const { date, humidity, temperature, wind } = forecast;
+  const formattedDate = new Date(date).toDateString();
+
+  return (
+    <div className="forecast-details">
+      <div>{formattedDate}</div>
+      <div>
+        <strong>Min temperature:</strong> {temperature.min} &deg;C
```

Comment:
Generally Id advise avoiding html tags like strong, instead use classnames to alter the style to your liking